Dockerfile

```diff
@@ -20,9 +20,7 @@ RUN bundle exec rails assets:precompile && rm -fr /app/log
 
 FROM $base_image
 
-ENV GOVUK_APP_NAME=finder-frontend PORT=3062
-
-HEALTHCHECK CMD curl --silent --fail localhost:$PORT/healthcheck/ready || exit 1
```

Comment:
Do you need to [move this to docker-compose](https://docs.docker.com/compose/compose-file/#healthcheck)? Or is it definitely not relevant for publishing-e2e-tests? (Does anything in the test fixtures rely on waiting for finder-frontend to become ready before sending request to it?)

Reply:
Oh yes i forgot to add that - i will add that now good catch.

I suspect not likely needed - but i i will add it in the docker compose